python/ray/data/grouped_dataset.py

```diff
@@ -247,14 +248,23 @@ def map_groups(
 
         Examples:
             >>> # Return a single record per group (list of multiple records in,
-            >>> # list of a single record out). Note that median is not an
-            >>> # associative function so cannot be computed with aggregate().
+            >>> # list of a single record out).
             >>> import ray
             >>> import pandas as pd
             >>> import numpy as np
+            >>> # Get median per group. Note that median is not an associative
+            >>> # function so cannot be computed with aggregate().
             >>> ds = ray.data.range(100) # doctest: +SKIP
             >>> ds.groupby(lambda x: x % 3).map_groups( # doctest: +SKIP
             ...     lambda x: [np.median(x)])
+            >>> # Get first value per group.
+            >>> ds = ray.data.from_items([ # doctest: +SKIP
```

Comment:
Can those SKIPs be removed? It looks they should be able to run without issue in case here.

Reply:
@jianoaix - I guess they can be removed. Just to follow what we have here to be safe. Wanted to get this in before branch cut as well. I can audit all SKIPs in file later as followup.